.github/workflows/packaging.yml

```diff
@@ -3,6 +3,18 @@ name: packaging
 on: [push, pull_request]
 
 jobs:
+  version_check:
+    runs-on: ubuntu-18.04
```

Comment:
> The ubuntu-18.04 environment is deprecated, consider switching to ubuntu-20.04(ubuntu-latest), or ubuntu-22.04 instead.

Let's use 20.04 disregarding the version used in other pipelines

Reply:
Done, now using `20.04` for all `version-check`  jobs